### PR TITLE
Apply patch for fixed seed with minor modifications.

### PR DESCRIPTION
**Pull request by simfleischman**
Mostly copied from
https://github.com/hedgehogqa/haskell-hedgehog/issues/201#issuecomment-441401915

### CODE REVIEW

**Comment by simfleischman** [comment]
Build is green. Let me know if you have any feedback.

**Review by moodmosaic — APPROVED** [review]
Thank you! Looks good 👍

**Comment by shapr** [comment]
How do I support getting this merged?

**Comment by moodmosaic** [comment]
@shapr thank you, we are looking into this right now. Chances are it might be merged soon.

File: hedgehog/src/Hedgehog/Internal/Config.hs [hunk 1]
```diff
@@ -107,6 +116,20 @@ detectColor =
...
+detectSeed =
+  liftIO $ do
+    menv <- lookupEnv "HEDGEHOG_SEED"
+    case menv of
+      Nothing ->
+        Seed.random
+      Just env ->
+        let
+          [value, gamma] =
+            read . Text.unpack <$> Text.splitOn (Text.pack " ") (Text.pack env)
```
Comment (jacobstanley) [inline hedgehog/src/Hedgehog/Internal/Config.hs:129]: this pattern match is non-exhaustive

Comment (jacobstanley) [reply hedgehog/src/Hedgehog/Internal/Config.hs:129]: going to fix this and push to the other PR

**Comment by jacobstanley** [comment]
#446 has been merged.

**Comment by jacobstanley** [comment]
Available in [hedgehog-1.1.1](https://hackage.haskell.org/package/hedgehog-1.1.1)

**Pull request closed without merging**